include/fluent-bit/flb_config.h

```diff
@@ -144,6 +144,16 @@ struct flb_config {
     void *http_ctx;           /* Monkey HTTP context    */
 #endif
 
+    /* There are two ways to use proxy in fluent-bit:
+         #1 Similar with http and datadog plugin, passing proxy directly to
```

Comment:
multiline comment still not set: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#multiline-comment

Reply:
Fixed.